streams/us.m3u

```diff
@@ -1747,7 +1747,7 @@ https://na-all23.secdn.net/pegstream3-live/play/65ea794b-dd82-41ce-8e98-a9177289
 #EXTINF:-1 tvg-id="UCTVUniversityofCalifornia.us" status="online",UCTV University of California (720p) [Not 24/7]
 https://59e8e1c60a2b2.streamlock.net/509/509.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="UNWebTV.us" status="error",UN Web TV (540p)
-https://bcliveunivsecure-lh.akamaihd.net/i/un150_A1_1@575439/master.m3u8
+https://cdnapi.kaltura.com/p/2503451/sp/250345100/playManifest/entryId/1_gb6tjmle/protocol/https/format/applehttp/a.m3u8
```

Comment:
Are you sure this is a UN Web TV stream? 

Here's what I see:
<img width="692" alt="image" src="https://user-images.githubusercontent.com/7253922/156530594-b8a05b6d-babe-471d-a0db-49d3d062d813.png">

and that's not what the channel's website is showing
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/7253922/156530877-de3964fd-117c-401a-a4e8-2030e5e60cbb.png">

Reply:
The link I gave is from the United Nations website, and it's written "24hr stream"

Reply:
United Nations meetings stream : https://cdnapi.kaltura.com/p/2503451/sp/250345100/playManifest/entryId/1_4ghd8fth/protocol/https/format/applehttp/a.m3u8